torchmdnet/scripts/train.py

```diff
@@ -81,6 +82,9 @@ def get_args():
     parser.add_argument('--distance-influence', type=str, default='both', choices=['keys', 'values', 'both', 'none'], help='Where distance information is included inside the attention')
     parser.add_argument('--attn-activation', default='silu', choices=list(act_class_mapping.keys()), help='Attention activation function')
     parser.add_argument('--num-heads', type=int, default=8, help='Number of attention heads')
+    
+    # TensorNet specific
+    parser.add_argument('--equivariance-invariance_group', type=str, default='O(3)', help='Equivariance and invariance group of TensorNet')
```

Comment:
I think that should be `'--equivariance-invariance-group'`?  With a hyphen rather than an underscore.